workers/social/lib/social/models/foreignauth.coffee

```diff
@@ -0,0 +1,131 @@
+jraphical = require 'jraphical'
+KodingError = require '../error'
+{ parseClient } = require './utils'
+
+module.exports = class JForeignAuth extends jraphical.Module
```

Comment:
since we dont use relationships, jraphical.Module no needed, bongo Base or Model is sufficient